### PR TITLE
LPS-176845 | Test Automation | Master

### PR DESCRIPTION
**Pull request by davisaints**
- [LPS-176845](https://issues.liferay.com/browse/LPS-176845)

### CODE REVIEW

**Comment by davisaints** [comment]
ci:test:sf

**Comment by davisaints** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 1]
```diff
@@ -2838,6 +2838,62 @@ definition {
...
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 176845");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Integer 1",
+				fieldType = "Integer");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:2860]: It's possible to create it via API.

**Review by andreefarias — CHANGES_REQUESTED** [review]
Nice, it's good for me.

Only have some requests, fix it, please.

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase [hunk 2]
```diff
@@ -2838,6 +2838,62 @@ definition {
...
+				fieldName = "customInteger1",
+				objectName = "CustomObject176845",
+				value = 15);
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject176845");
+
+			Search.searchCP(searchTerm = 30);
+		}
+
+		task ("Then: The Formula Builder is not searchable") {
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase:2890]: `task ("Then: The Formula Field is not searchable") {`

**Comment by davisaints** [comment]
ci:test:sf

**Pull request closed without merging**